administrator/components/com_joomgallery/views/config/view.html.php

```diff
@@ -214,6 +214,11 @@ public function display($tpl = null)
 
     JText::script('COM_JOOMGALLERY_CONFIG_GS_PD_ALERT_THUMBNAIL_PATH_SUPPORT');
 
+    // Additional Tabs added by plugins
+    $this->event  = new stdClass();
+    $addTabs  = $this->_mainframe->triggerEvent('onJoomAfterDisplayTabs', array(_JOOM_OPTION.'.config',''));
+    $this->event->afterDisplayTabs = trim(implode('', $addTabs));
```

Comment:
According to https://www.joomgalleryfriends.net/plugin-events.html#onJoomAfterDisplayTabs "$addTabs" has the type "string" but `implode('', $addTabs)` needs an array.